utils/common/src/main/java/org/apache/brooklyn/util/http/executor/UsernamePassword.java

```diff
@@ -16,28 +16,14 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.brooklyn.util.http.executor;
-
-import com.google.common.annotations.Beta;
 
-@Beta
-public class UsernamePassword implements Credentials {
-    private final String username;
-    private final String password;
+package org.apache.brooklyn.util.http.executor;
 
+/**
+ * @deprecated since 1.0.0 use {@link org.apache.brooklyn.util.http.auth.UsernamePassword} instead.
```

Comment:
Minor comment I thought of (after merging!) is that it's worth saying why we're leaving this class around (for persisted state backwards compatibility, as well as just normal deprecation policy). We could add something to https://github.com/apache/brooklyn-server/blob/master/core/src/main/resources/org/apache/brooklyn/core/mgmt/persist/deserializingClassRenames.properties, to support future deletion of this class (I don't think we want it deleted without adding a line there).